templates/collections/explorer_index_page.html

```diff
@@ -5,13 +5,12 @@
 {% block content %}
     {% include 'includes/generic-intro--dark.html' %}
 
-    <div class="container">
-        <div class="row">
-            <div class="col-md-12" id="content">
-                {% for block in page.body %}
-                    {% include_block block %}
-                {% endfor %}
-            </div>
-        </div>
+    <div class="container mt-4">
+       <ul class="row card-group">
```

Comment:
Similar to the comments above, we should avoid mixing BEM and Bootstrap on the same element.